src/icons/blockContentIcon.tsx

```diff
@@ -0,0 +1,53 @@
+/* THIS FILE IS AUTO-GENERATED – DO NOT EDIT */
+
+import {forwardRef} from 'react'
+
+/**
+ * @public
+ */
+export const BlockContentIcon = forwardRef(function BlockContentIcon(
+  props: React.SVGProps<SVGSVGElement>,
+  ref: React.Ref<SVGSVGElement>
+) {
+  return (
+    <svg
+      data-sanity-icon="block-content"
+      width="1em"
+      height="1em"
+      viewBox="0 0 26 25"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      ref={ref}
+      {...props}
+    >
+      <path
+        fillRule="evenodd"
+        clipRule="evenodd"
+        d="M21.0126 7.59999L11.0126 7.59999V6.39999L21.0126 6.39999V7.59999Z"
+        fill="#101112"
```

Comment:
All generated icon components should use `currentColor`

This should be possible by adjusting the colour of the source SVGs to `#121923`, after which the icon build process will substitute those values with `currentColor`, see: https://github.com/sanity-io/icons/blob/57194308de5626b392b7aa8e6ccbb1bc1ae7173b/scripts/generate.ts#L80-L83